moveit_core/distance_field/src/distance_field.cpp

```diff
@@ -428,7 +428,7 @@ void DistanceField::getPlaneMarkers(PlaneVisualizationType type, double length,
           continue;
         }
         double dist = getDistance(x, y, z);
-        int last{ plane_marker.points.size() };
+        int last = plane_marker.points.size();
```

Comment:
This is still a narrowing conversion, it is just implicit when you use operator=. It would be better to change this to not be a narrowing conversion. There are a couple ways you can do that. The first is to use auto:

```c++
auto last{ plane_marker.points.size() };
```
This will cause the type of last to be whatever the type returned by the size() function is.
The second would be to use an unsigned type like `size_t` (this is likely the type returned by size()):

```cpp
size_t last{ plane_marker.points.size() };
```

Reply:
Thank you for the suggestions! I will fix this.